domain/schema/model/sql/0019-application.sql

```diff
@@ -28,6 +28,20 @@ CREATE TABLE application (
 CREATE UNIQUE INDEX idx_application_name
 ON application (name);
 
+-- application_subordinate links subordinate applications to their
+-- principals. A subordinate application can have one principal. A
+-- principal application have more than one subordinate.
+CREATE TABLE application_subordinate (
```

Comment:
This should be a view not a table. You know this when a relation is made.

Reply:
Yeah, it's something like all subordinate applications joined to related applications where the relation is container scoped. Not a crazy query.

Reply:
done